docs/anchoring-assets/anchor-platform/events/integration.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: "Integration"
+sidebar_position: 20
+---
+
+This guide will walk you through integrating with the event service to start receiving events. The event service currently only supports Apache Kafka as the backend message broker.
+
+It assumes familiarity with Kafka and will not cover how to set up a Kafka cluster.
+
+## Requirements
+
+Anchor Platform will send events to the `TRANSACTION` Kafka topic. The event service will consume events from this topic and send them to the appropriate endpoints.
+
+## Configuration
+
+First, the event processor needs to be configured in the `event_processor` section of the Anchor Platform Configuration file.
+
+<CodeExample>
+
+```yaml
+event_processor:
+  client_status_callback:
+    enabled: true
+  callback_api_request:
+    enabled: true
+```
+
+</CodeExample>
+
+This will enable the event processor to start processing events from `TRANSACTION` topic. In this example, the event processor will send events to client and business server callback endpoints.
+
+Next, the event service's Kafka producer need to be configured using the `event.queue` section of the configuration file. The following is an example of how to configure the event service's Kafka producer:
+
+<CodeExample>
+
+```yaml
```

Comment:
In all other places we use env variables for configs. 
I know it's easier with yaml but I think for at least v2 of the document we should use env everywhere for consistency